content/blog/2021-11-30-international-rse-survey.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Take the 2021 international RSE survey: help us improve the RSE community"
+
+[extra]
+
+author = "Patric Holmvall, Radovan Bast, Richard Darst" 
++++
+
+
```

Comment:
In general I like the idea of summaries, but in this case I think it will have the opposite effect of what is desired: most people who end up on this blog post will probably do so via an email/tweet, which already contains a shorter version? Another summary/short version might just confuse/bloat the text, or what do we think?

Reply:
Hm but what about the 'browsers' ? 
I think the summary text now is still actually too long. Will give it a quick rewrite. And then you can decide to keep or kick ;)

Reply:
Sorry work calls, cannot add it now. PLease merge it without and if I find time I will send a follow up PR.

Reply:
You're right about the browsers. I think I'd not be against the summary if it wasn't for the fact that the same sentences keep repeating so many times: someone landing on the page from e.g. an email will read the same starting sentences 3 times.

Reply:
Agree that the current suggested text does not fit this best. I suggest to merge without the intro text for now, to have it on the webpage